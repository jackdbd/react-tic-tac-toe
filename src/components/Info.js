/**
 * Info module.
 * @module /components/Info
 */
import React from "react";
import styled from "styled-components";

const Div = styled.div`
  background-color: #68838b;
  margin-left: 20px;
`;

const DivStatus = styled.div`
  font: 24px "Lobster";
  background-color: steelblue;
  margin-bottom: 20px;
`;

const Ol = styled.ol`
  padding-left: 30px;
  padding-right: 10px;
`;

const Info = props => {
  const { status, moves } = props;
  return (
    <Div>
      <DivStatus>{status}</DivStatus>
      <Ol>{moves}</Ol>
    </Div>
  );
};

export default Info;
