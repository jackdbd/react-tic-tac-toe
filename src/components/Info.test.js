import React from "react";
import ReactDOM from "react-dom";
import Info from "./Info";

describe("Info component", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Info />, div);
  });
  it("renders properly", () => {
    expect(<Info />).toMatchSnapshot();
  });
});
