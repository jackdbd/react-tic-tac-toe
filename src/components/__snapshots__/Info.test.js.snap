// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info component renders properly 1`] = `<Info />`;
